R/CallablePythonCode.R

```diff
@@ -176,15 +176,19 @@ PythonCodeClass <- R6::R6Class( # nolint
 #'
 #' @examples
 #' \dontrun{
-#' library(scda)
 #' library(reticulate)
 #' library(magrittr)
 #'
 #' # mutate dataset object
 #'
-#' ADSL <- synthetic_cdisc_data("latest")$adsl
-#'
-#' x <- scda_cdisc_dataset_connector("ADSL", "adsl")
+#' ADSL <- teal.data::rADSL
+#' pull_adsl <- function(ADSL) ADSL <- teal.data::rADSL
+#' x <- dataset_connector(
+#'   dataname = "ADSL",
+#'   pull_callable = callable_function(fun = pull_adsl),
+#'   keys = get_cdisc_keys("ADSL"),
+#'   label = "Example label"
+#' )
```

Comment:
I think, you can leverage `example_cdisc_data` and `fun_dataset_connector` here and reuse in this 

```suggestion
#' random_data_connector <- function(dataname) {
#'   fun_dataset_connector(
#'     dataname = dataname,
#'     fun = example_cdisc_data,
#'     fun_args = list(dataname = dataname),
#'   )
#' }
#' ADSL <- random_data_connector(dataname = "ADSL")
```